appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts

```diff
@@ -35,12 +41,21 @@ describe('COMPOSE EMAIL: ', () => {
         MenuBarScreen.clickMenuIcon();
         MenuBarScreen.clickInboxButton();
 
+        // Add first contact
         InboxScreen.clickCreateEmail();
 
-        NewMessageScreen.setAddRecipientByName(contactName, contactEmail);
-        NewMessageScreen.checkAddedRecipient(contactEmail);
+        NewMessageScreen.setAddRecipientByName(firstContactName, firstContactEmail);
+        NewMessageScreen.checkAddedRecipient(firstContactEmail);
         NewMessageScreen.clickBackButton();
 
+        // Add second contact
+        InboxScreen.clickCreateEmail();
```

Comment:
Could have added a second recipient on the same screen. It's a minor detail, but would speed up the test a little bit.